### PR TITLE
UI improvements for account recovery flow

### PR DESCRIPTION
**Pull request by anikarni**
TODO:

- [ ] Translate to portuguese
- [x] Add progress bar
- [x] Fix desktop version
- [x] Add user recovery code form
- [x] Make password not show
- [ ] Move forms to own directories
- [ ] Make url public

Related with #932 

acc-rec2

### CODE REVIEW

File: web-ui/src/account_recovery/forms/user_recovery_code_form.js [hunk 1]
```diff
@@ -20,25 +20,39 @@ import { translate } from 'react-i18next';
...
+        src='/public/images/account-recovery/codes.svg'
+        alt=''
+      />
+      <p>{t('account-recovery.user-form.description')}</p>
+    </div>
     <InputField name='admin-code' label={t('account-recovery.user-form.input-label')} />
     <SubmitButton buttonText={t('account-recovery.user-form.button')} />
+    <BackLink
+      text={t('account-recovery.back')}
+      onClick={previous} onKeyDown={previous}
```
Comment (FrailWords) [inline web-ui/src/account_recovery/forms/user_recovery_code_form.js]: Is there a specific case for which we have a 'keyDown' handler?

File: web-ui/src/backup_account/backup_email/backup_email.js [hunk 2]
```diff
@@ -72,12 +74,10 @@ export class BackupEmail extends React.Component {
...
-          <div className='link-content'>
-            <a href='/' className='link'>
-              <i className='fa fa-angle-left' aria-hidden='true' />
-              <span>{t('back-to-inbox')}</span>
-            </a>
-          </div>
+          <BackLink
+            href='/'
+            text={t('back-to-inbox')}
+          />
```
Comment (FrailWords) [inline web-ui/src/backup_account/backup_email/backup_email.js]: Should we also this component on the 'Confirmation' page of the backup-account flow?

File: web-ui/src/account_recovery/forms/user_recovery_code_form.js (hunk 1 above)
Comment (anikarni) [inline web-ui/src/account_recovery/forms/user_recovery_code_form.js]: Yes! Keyboard enter! For accessibility and for those developers that refuse to use the mouse ( ✋ )

File: web-ui/src/backup_account/backup_email/backup_email.js (hunk 2 above)
Comment (anikarni) [inline web-ui/src/backup_account/backup_email/backup_email.js]: Yeeeeees! I didn't know there was a link there. Absolutely. Thanks!

**Pull request merged**